### ISSUE TITLE
[communities] can't leave region communities

### ISSUE THREAD

**Issue by bakeiro** [Couchers-org/couchers#4093]
### Me as user I want

To decide in which communities I am and which ones I'm not

### current situation:

Not possible to leave a community (image attached)

![image](https://github.com/Couchers-org/web-frontend/assets/16777161/721ff357-0083-4a13-bece-3a48bf0df544)

### notes:
In this picture you can see I belong to the africa/europe/middle east community, which IMHO doesn't make any sense at all to belong.
I think this because is pretty big, general, and I see event of middle east and Africa which of course I can't join, and also the discussions in this community would make more sense if the community would be smaller

**Comment by jesseallhands** [comment]
This is by design. You can't leave a community you're a part of and since `Europe, Africa, and Middle East` is the parent community of the `Spain` community then you can't leave `Europe, Africa, and Middle East` without first leaving `Spain`. 

So perhaps we should remove the "leave community" button if it is not possible for a user to leave it. 

To solve the other problem you mentioned: If a user doesn't want to see events or discussions from parent communities (perhaps they only want to see them from their own country or city) then they should be able to restrict that. That is something that would need to be discussed and designed, but I see it as a very low priority right now.

**Comment by bakeiro** [comment]
> This is by design. You can't leave a community you're a part of and since `Europe, Africa, and Middle East` is the parent community of the `Spain` community then you can't leave `Europe, Africa, and Middle East` without first leaving `Spain`.
> 
> So perhaps we should remove the "leave community" button if it is not possible for a user to leave it.
> 
> To solve the other problem you mentioned: If a user doesn't want to see events or discussions from parent communities (perhaps they only want to see them from their own country or city) then they should be able to restrict that. That is something that would need to be discussed and designed, but I see it as a very low priority right now.

yeah... in this very specific issue, I think the problem is the `Europe, Africa and Middle East` community, I still don´t understand it, it's just to big... I don't think it makes sense... also being able to leave communities that are the parent community of a smaller one (which I belong) doens't sound bad to me, what if there are so many events in this very big community in the future? and I want to filter all that 'noise' and focus only in smaller communities? I think should be possible (why not?)

**Comment by bakeiro** [comment]
I did a test, I just leave `Madrid` and `Spain` community, buuut I can't leave `Europe, Africa and Middle East`, so I don't think it's something related with leaving a community which is the parent of other community which I belong
![image](https://github.com/user-attachments/assets/fbc36bc2-aba6-494f-b593-731052c82742)

**Comment by jesseallhands** [comment]
I think this has something to do with how the community parent/child structure was designed by Aapeli. 

It sounds like the main problem is being able to unsubscribe from events/discussions from a larger (parent) community that the user may not be interested in. That is an issue that could and should be addressed without needing to fundamentally change the backend.

**Comment by nabramow** [comment]
Closing as we decided this is a feature, not a bug. The changes to remove discussions and events from the region communities should also decrease the need for this.
